test/jdk/java/awt/Checkbox/CheckboxBoxSizeTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2001, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.Checkbox;
+import java.awt.FlowLayout;
+import java.awt.Frame;
+import java.awt.Panel;
+
+/*
+ * @test
+ * @bug 4410522
+ * @requires (os.family == "windows")
+ * @summary The box size of the Checkbox control should be the same as
+ *          in Windows native applications.
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual CheckboxBoxSizeTest
+ */
+
+public class CheckboxBoxSizeTest {
+    private static final String INSTRUCTIONS = """
+            This test must be run at UI Scale of 100% OR 150% or greater.
```

Comment:
@prsadhuk Tested at different UI Scales on Windows and "the box" doesn't seem to scale properly and doesn't match windows native app (used Word -> Font settings to check). 
I think it is product bug. These tests weren't problemlisted so I may need to create a new JBS bug and problemlist this test and probably CheckboxIndicatorSizeTest.

Reply:
Looks to be similar to [JDK-8233068](https://bugs.openjdk.org/browse/JDK-8233068)

Reply:
@prsadhuk Thanks for linking the issue. Yes it looks to be similar.
Unlike XCheckboxPeer.java, WCheckboxPeer.java does not have paintCheckbox(), it is probably done somewhere else for windows.

Reply:
Problemlisted the tests - CheckboxNullLabelTest.java, CheckboxIndicatorSizeTest.java, CheckboxBoxSizeTest.java on windows. [JDK-8340870](https://bugs.openjdk.org/browse/JDK-8340870)